x/tokenfactory/keeper/before_send.go

```diff
@@ -99,6 +99,11 @@ func (k Keeper) callBeforeSendListener(ctx context.Context, from, to sdk.AccAddr
 				return err
 			}
 
+			// Do not invoke hook if denom is not whitelisted
+			if err := k.AssertIsHookWhitelisted(c, coin.Denom, cwAddr); err != nil {
+				return nil
+			}
+
```

Comment:
Just confirming we are good with this behavior? @zavgorodnii @sotnikov-s

Reply:
even though we remove all nonwhitelisted hooks during migration, I still think it's more appropriate to return an error here. we have two options here in case of an unauthorised hook: 1) we completely ignore all the hooks code (return nil here breaking the method execution) or 2) we return error here explicitly making all bank operations with unauthorised hooks fail. I prefer the second one

Reply:
if we still stick to the first option described above, we should replace `return nil` with `continue` because the `return nil` happens in a loop of handling a bank operation that possibly contains multiple assets. if just return, we break the loop and ignore the rest of the assets

Reply:
Spoke with Andrei about this. This line is really only here as an emergency override in case something is wrong with a hook and we want to use the sub-dao to turn it off immediately. This is an extreme case and just a stop-gap before we would do an emergency upgrade to remove the problematic hook. In this scenario we do not want the offending hook to be able to block sends. 
Agree on switching to `continue` though.

Reply:
please also add an error logging here